sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Models/DatasourceCredentialsTests.cs

```diff
@@ -16,9 +17,14 @@ namespace Azure.AI.MetricsAdvisor.Tests
 {
     public class DatasourceCredentialsTests : MockClientTestBase
```

Comment:
Non-live tests to make sure we're passing the secret to the service. The service never returns the secrets back to us, so there's no way to validate it live.